lib/src/scss-shared/_nx-input-helpers.scss

```diff
@@ -3,19 +3,16 @@
 }
 
 @mixin focusable-toggle-input($focus-border-selector) {
-  position: relative;
-
   .nx-radio__input, .nx-checkbox__input {
-    position: absolute;
-    z-index: -1;
-
     &:focus {
       // less-nice IE11 focus style because IE11 doesn't support :focus-within
+      height: 100%;
+      margin-top: 5px;
       margin-right: $nx-spacing-xxs;
       outline: 1px solid $nx-color-blue-44;
       opacity: 1;
       position: static;
-      height: 100%;
+      vertical-align: top;
```

Comment:
Consider using a number for `vertical-align` (coincidentally 5px still seems to work) rather than this plus a separate margin.

Reply:
> Consider using a number for `vertical-align` (coincidentally 5px still seems to work) rather than this plus a separate margin.

I always forget I can use a value for vertical-align, thanks for the reminder: https://github.com/sonatype/sonatype-react-shared-components/pull/86/commits/74308db5d3ddf7206ab167c431feba89dc373d74